server/game/cards/04.2-TE/KuniYori.js

```diff
@@ -1,10 +1,27 @@
 const DrawCard = require('../../drawcard.js');
 
 class KuniYori extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (this.game.currentConflict && this.game.currentConflict.hasElement('earth')),
```

Comment:
I'd rather you uses: `condition: () => this.game.isDuringConflict('earth')`
just in case we change the way conflicts are structured in the future